app/views/venues/show.html.erb

```diff
@@ -4,4 +4,5 @@
   <%= link_to @venue.website, @venue.website, class: "website", target: "_blank" %>
   <p class="venue-address"><%= @venue.address %></p>
   <p class="venue-capacity">Capacity: <%= @venue.capacity %></p>
+  <p class="venue-t-accessible"><%= @venue.display_t_accessible %></p>
```

Comment:
We were attempting to remove logic from the view and place it in the model. Can we put the whole HTML line in the model, can we get away with just returning the blank string like we have, or should we just give up and put the logic in the view?